app/views/users/edit.html.erb

```diff
@@ -120,6 +120,7 @@
       <%= link_to t('.cancel', default: t('helpers.links.cancel')),
                 user_path(@user), class: 'btn btn-default' %>
       <%= t('.unhappy') %>? <%= link_to t('.cancel_this_account'), user_path(@user), method: :delete %>.
+      <%= back_btn root_path %>
```

Comment:
@Oluwoye I deleted `users/edit_profile`, since this file got merged into `users/edit` (as you see here). Is this one linge the only change you made to `users/edit_profile?`

Reply:
yes